Add Agriculture

beaorn opened the pull request:
<!-- Congrats on creating an Awesome list! 🎉 -->


<!-- Please fill in the below placeholders -->

**https://github.com/beaorn/awesome-agriculture**

**Awesome Agriculture is a collection of open source and/or free tools, software and APIs related to Agriculture. It also includes open source API clients and tools to work with proprietary APIs and data formats. This list aims to remedy the fact that there isn't really a good place to find out about ag and farming related open source software projects easily in one place.**


### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:


## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

sindresorhus commented:
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

sindresorhus commented:
Cool idea!

beaorn commented:
Thanks @sindresorhus  and sorry about not following all the guidelines properly on first submit. 

I just updated the list to fix the following issues, I'm hoping this cleaned everything up:
- Table of Contents section heading fixed as specified in guidelines
- Made the list description more succinct
- Fixed consistency issues in list item formatting, reduced some of the description lengths, added periods at ends of descriptions
- Removed unnecessary sub-sections

sindresorhus commented on readme.md:
```diff
@@ -577,6 +577,7 @@
 - [Visual Regression Testing](https://github.com/mojoaxel/awesome-regression-testing#readme) - Ensures changes did not break the functionality or style.
 - [Theravada](https://github.com/johnjago/awesome-theravada#readme) - Teachings from the Theravada Buddhist tradition.
 - [inspectIT](https://github.com/inspectit-labs/awesome-inspectit#readme) - Open source Java app performance management tool.
+- [Agriculture](https://github.com/beaorn/awesome-agriculture#readme) - Agriculture related open source software, tools and APIs.
```
The description should be about the topic, not your list.

For example:

> Open source technology for farming and gardening.

sindresorhus commented:
> Includes a succinct description of the project/theme at the top of the readme. (Example)

⬆️ Should describe agriculture.


---

Linkify `Agriculture` in `A curated list of awesome open source tools, software and APIs related to Agriculture.` to https://en.wikipedia.org/wiki/Agriculture

---

Any related communities you could link to? Like reddit, forums, etc?

sindresorhus commented:
```
❯ awesome-lint https://github.com/beaorn/awesome-agriculture
✖ Linting

  README.md:22:1
  ✖    1:1   Missing License section                                        remark-lint:awesome/license
  ✖    1:1   Missing or invalid Table of Contents                           remark-lint:awesome/toc
  ✖    9:3   Invalid list item link URL                                     remark-lint:awesome/list-item
  ✖   10:3   Invalid list item link URL                                     remark-lint:awesome/list-item
  ✖   11:3   Invalid list item link URL                                     remark-lint:awesome/list-item
  ✖   12:3   Invalid list item link URL                                     remark-lint:awesome/list-item
  ✖   13:3   Invalid list item link URL                                     remark-lint:awesome/list-item
  ✖   14:3   Invalid list item link URL                                     remark-lint:awesome/list-item
  ✖   15:3   Invalid list item link URL                                     remark-lint:awesome/list-item
  ✖   16:3   Invalid list item link URL                                     remark-lint:awesome/list-item
  ✖   17:3   Invalid list item link URL                                     remark-lint:awesome/list-item
  ✖   18:3   Invalid list item link URL                                     remark-lint:awesome/list-item
  ✖   22:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   26:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   27:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   28:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   29:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   30:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   31:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   32:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   33:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   34:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   38:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   42:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   43:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   43:3   Link to https://github.com/woodardjoshua/Ag-Analytics is dead  remark-lint:no-dead-urls
  ✖   44:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   45:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   46:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   47:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   51:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   52:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   53:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   54:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   54:68  List item description must start with valid casing             remark-lint:awesome/list-item
  ✖   55:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   56:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   57:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   58:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   58:96  List item description must start with valid casing             remark-lint:awesome/list-item
  ✖   59:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   60:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   61:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   62:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   63:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   64:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   68:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   69:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   70:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   71:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   77:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   81:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   85:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   86:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   86:59  List item description must start with valid casing             remark-lint:awesome/list-item
  ✖   90:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   91:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   92:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   93:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   94:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   98:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖   99:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖  103:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖  104:1   Marker style should be -                                       remark-lint:unordered-list-marker-style
  ✖  105:1   Marker style should be -                                       remark-lint:unordered-list-marker-style

  65 errors
```

https://github.com/sindresorhus/awesome-lint

sindresorhus commented:
Can you fix the merge conflict?

sindresorhus commented:
> * **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)

sindresorhus commented:
https://github.com/sindresorhus/awesome/pull/1316/files#r205952906

beaorn commented:
@sindresorhus merge conflict is fixed and verified awesome-lint no longer returns any errors, hopefully got everything fixed. Thanks!

sindresorhus commented:
There's still https://github.com/sindresorhus/awesome/pull/1316#discussion_r205952906

beaorn commented:
> There's still [#1316 (comment)](https://github.com/sindresorhus/awesome/pull/1316#discussion_r205952906)

Sorry, that should be corrected now

beaorn commented:
If there is anything on my end lacking to prevent this from getting merged let me know and I will fix it asap. Thanks!

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1102226507353620480
